### PR TITLE
Updates the return values to reflect what the product actually returns.

### PR DESCRIPTION
**Pull request by gem-neo4j**
From [CIP-49](https://docs.google.com/document/d/1PHnAYYAe9U97YtKpZyaMObhhjEferE_Ynw5gYhRiXTs/edit#heading=h.lzdax825k5wi)

### CODE REVIEW

**Comment by gem-neo4j** [comment]
[Log function wrong result](https://trello.com/c/KoW6aBNE/636-log-function-wrong-result)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-cypher-754.surge.sh

File: modules/ROOT/pages/functions/mathematical-logarithmic.adoc [hunk 1]
```diff
@@ -89,6 +89,7 @@ e(expression)
 |===
 
 | `exp(null)` returns `null`.
+| `exp()` returns `Infinity` when the return value is greater than the largest `FLOAT` value.
```
Comment (Lojjs) [inline modules/ROOT/pages/functions/mathematical-logarithmic.adoc]: If the docs reviewer agrees, maybe we should add a note here to what the largest FLOAT values is (Java double.MAX_VALUE I believe)

File: modules/ROOT/pages/functions/mathematical-logarithmic.adoc [hunk 2]
```diff
@@ -152,7 +153,8 @@ log(expression)
 |===
 
 | `log(null)` returns `null`.
-| `log(0)` returns `null`.
+| `log(0)` returns `-Infinity`.
+| If (`expression` < 1), then (`log(expression)`) returns `NaN`.
```
Comment (Lojjs) [inline modules/ROOT/pages/functions/mathematical-logarithmic.adoc]:
```suggestion
| If (`expression` < 0), then (`log(expression)`) returns `NaN`.
```

File: modules/ROOT/pages/functions/mathematical-logarithmic.adoc [hunk 3]
```diff
@@ -217,7 +219,8 @@ log10(expression)
 |===
 
 | `log10(null)` returns `null`.
-| `log10(0)` returns `null`.
+| `log10(0)` returns `-Infinity`.
+| If (`expression` < 1), then (`log10(expression)`) returns `NaN`.
```
Comment (Lojjs) [inline modules/ROOT/pages/functions/mathematical-logarithmic.adoc]:
```suggestion
| If (`expression` < 0), then (`log10(expression)`) returns `NaN`.
```

Comment (Lojjs) [reply modules/ROOT/pages/functions/mathematical-logarithmic.adoc]: We might want to rewrite sqrt in the same way to align, right now it says

sqrt(\<any negative number\>) returns NaN

**Review by Lojjs — CHANGES_REQUESTED** [review]
Two typos and two small suggestions

**Review by Lojjs — APPROVED** [review]

**Review by JPryce-Aklundh — APPROVED** [review]
lgtm 👍

**Pull request merged**